util/chplenv/chpl_atomics.py

```diff
@@ -38,6 +38,8 @@ def get(flag='target'):
                     atomics_val = 'intrinsics'
                 elif version >= CompVersion('4.1') and not platform_val.endswith('32'):
                     atomics_val = 'intrinsics'
+            elif compiler_val == 'aarch64-gnu':
+                atomics_val = 'cstdlib'
```

Comment:
Some of the intrinsics code doesn't work on the ARM cross-gcc.  I could code around it in intrinsics, adding some cruft, but it's cleaner to just specify cstdlib here.  The Cray compiler is also missing some ARM intrinsic support.